docs/install/integrations.md

```diff
@@ -127,20 +129,26 @@ dependencies {
     // ktlint will pick them up
 }
 
-task ktlint(type: JavaExec, group: "verification") {
+tasks.register("ktlintCheck", JavaExec) {
+    group = "verification"
     description = "Check Kotlin code style."
     classpath = configurations.ktlint
-    mainClass.set("com.pinterest.ktlint.Main")
-    args "src/**/*.kt"
+    mainClass = "com.pinterest.ktlint.Main"
+    args "src/**/*.kt", "!**/build/**"
     // see https://pinterest.github.io/ktlint/install/cli/#command-line-usage for more information
 }
-check.dependsOn ktlint
 
-task ktlintFormat(type: JavaExec, group: "formatting") {
+tasks.named("check") {
+    dependsOn tasks.named("ktlintCheck")
+}
+
+tasks.register("ktlintFormat", JavaExec) {
+    group = "formatting"
     description = "Fix Kotlin code style deviations."
     classpath = configurations.ktlint
-    mainClass.set("com.pinterest.ktlint.Main")
-    args "-F", "src/**/*.kt"
+    mainClass = "com.pinterest.ktlint.Main"
+    jvmArgs "--add-opens=java.base/java.lang=ALL-UNNAMED"
```

Comment:
I'm not sure why there are 2 args in Maven integration but 1 in Gradle's,

https://github.com/pinterest/ktlint/blob/8757eb6b99c66d83d2d160545eb0a1a45f62fc4c/docs/install/integrations.md?plain=1#L25-L26

Reply:
I just see `--add-opens java.base/java.lang=ALL-UNNAMED` in [gantsign/ktlint-maven-plugin](https://github.com/gantsign/ktlint-maven-plugin#using-java-17-and-later), and this should be used >= Java 16?

Reply:
After some digging around, I found only [this reference](https://github.com/pinterest/ktlint/issues/1195#issuecomment-895034955) in the issue tracker but that didn't lead to any code change. The `--add-opens` for `java.base/java.util` was added in this [commit](https://github.com/pinterest/ktlint/commit/8dd55c4718edb55cb1f34e4d422a612ec2424c6c#diff-28fa7445d504e87ed9fcf4e359d8d6858a9ca1ea6a12cc5b917b9a8e644ca858R130). Most likely this was based on the example in [Gradle documentation](https://docs.gradle.org/7.5/userguide/upgrading_version_7.html#removes_implicit_add_opens_for_test_workers). I have reused the change in the `CommandLineTestRunner` and added both `--add-opens` to the documentation you mentioned above. I will create a PR to validate whether all keeps working without the `add-opens` for `java.base/java.util`.

Reply:
Yeah, I believe 2 args are only needed in tests, but 1 arg is for formatting tasks.

Reply:
Yes, lets verify and fix in #1886